Interface.C

```diff
@@ -31,6 +30,23 @@ preciceAdapter::Interface::Interface(
                           "warning"));
     }
 
+    if (locationsType == "faceCenters" || locationsType == "faceCentres")
+    {
+        locationType_ = LocationType::faceCenters;
+    }
+    else if (locationsType == "faceNodes")
+    {
+        locationType_ = LocationType::faceNodes;
+    }
+    else
+    {
+        adapterInfo("Interface points location type \""
+                    "locations = "
+                        + locationsType + "\" is invalid.",
+                    "error");
```

Comment:
Did you try if this error is triggered now? Maybe we need the `error-deferred`, because this step happens during the configuration.

Reply:
I tried both, and both worked IIRC, I will try once more.